jmdaemon/jmdaemon/onionmc.py

```diff
@@ -583,19 +583,23 @@ def try_to_connect(self) -> None:
 
 class OnionDirectoryPeer(OnionPeer):
     delay = 4.0
+
     def try_to_connect(self) -> None:
         # Delay deliberately expands out to very
         # long times as yg-s tend to be very long
         # running bots:
         self.delay *= 1.5
         if self.delay > 10000:
             log.warn("Cannot connect to directory node peer: {} "
-                     "after 20 attempts, giving up.".format(self.peer_location()))
+                     "after 20 attempts, giving up.".format(
+                         self.peer_location()))
             return
-        try:
-            self.connect()
-        except OnionPeerConnectionError:
-            reactor.callLater(self.delay, self.try_to_connect)
+        #try:
+        log.info(f"Going to reattempt connection to {self.peer_location()} in "
+                 f"{self.delay} seconds.")

```

Comment:
```suggestion
        log.info(f"Going to reattempt connection to {self.peer_location()} in "
                 f"{self.delay + jitter} seconds.")
```
Should we add the `jitter` here? We could create a variable `real_delay` or something.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1436/commits/f19c0f086f4bcbd9585dda93c870c60430005549